app/validators/form_password_validator.rb

```diff
@@ -5,25 +5,6 @@ module FormPasswordValidator
     validates :password,
               presence: true,
               length: Devise.password_length,
-              confirmation: true,
-              if: :password_required?
-  end
-
-  private
-
-  def password_required?
-    new_user? || user_resetting_password? || user_updating_password?
-  end
-
-  def new_user?
-    @user.encrypted_password.blank?
-  end
-
-  def user_resetting_password?
-    @user.reset_password_token.present?
-  end
-
-  def user_updating_password?
-    password.present?
+              confirmation: true
```

Comment:
I don't think this is needed. It would be needed if we wanted the user to type their password in twice.